supereeg/model.py

```diff
@@ -366,6 +359,8 @@ def update(self, data, inplace=True):
         else:
             m1 = Model(self)
 
+        assert m1.meta['stable']==True, 'solution unstable'
```

Comment:
expand on this error message: I suggest changing to "Model is numerically unstable; cannot update model"

Instead of throwing an error, we could instead print a warning if `m1.meta['stable']` is `False`, and then proceed anyway.  (E.g. "Model is unstable; results may be unreliable.")